app-ios/Sources/SessionFeature/SessionAssetsView.swift

```diff
@@ -0,0 +1,42 @@
+import appioscombined
+import SwiftUI
+import Strings
```

Comment:
Please add dependency about `Assets` , `Strings` and `Theme` to `Package.swift` 🙏🏼

Reply:
Done in  https://github.com/DroidKaigi/conference-app-2022/commit/cc4616edea505764c7b5ae921b873d0a817c1132